### PR TITLE
RWD - Desktop Version

### PR DESCRIPTION
**Pull request by altontonn**
Media query for mobile and desktop portfolio

### CODE REVIEW

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @altontonn, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The PR title and description look good and professional 👍
- No linter errors 👍
- Descriptive commit message 👍

## Required Changes ♻️
_Check the comments under the review._
### There is no provision for inline comments because major files were committed in another PR
- [x] Kindly make alignments to the app according to the images I have shared below;

### Yours
Yours

### Figma
Figma

### Yours
desk3
### Figma
desk3_original

### Yours
desk2

### Figma
desk2_original

### Yours
desk1

### Figma
desk1_original


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.css [hunk 1]
```diff
@@ -1,11 +1,524 @@
...
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    letter-spacing: 0.03em;
+    color: #fff;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
```
Comment (leonardodiasb) [inline main.css:315]: - [OPTIONAL] 'See Project' buttons should appear on hover.
image

File: main.css [hunk 2]
```diff
@@ -1,11 +1,524 @@
-body {
-    background-color: #E5E5E5;
-}
-.mobile{
-    width: 375px;
+@media screen and (min-width: 375px){ 
+  body {
+      margin: 0;
+      width: 1350px;
+  }
```
Comment (leonardodiasb) [inline main.css:5]: - [OPTIONAL] Kindly don't use fixed widths for setting the body of the document. This is causing blank space on the side of your app on some screen sizes.
image

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @altontonn  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._






## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

**Pull request merged**